lib/puppet/provider/key_bundle/katello_ssl_tool.rb

```diff
@@ -2,6 +2,13 @@
 
 Puppet::Type.type(:key_bundle).provide(:katello_ssl_tool, :parent => Puppet::Provider::KatelloSslTool::CertFile) do
 
+  def exists?
+    return false unless File.exists?(resource[:path])
+    return false unless File.exists?(privkey_source_path)
+    return false unless File.exists?(pubkey_source_path)
+    expected_content_processed == current_content
```

Comment:
Is this really part of `exists?` Isn't this more `insync?` behavior?

Reply:
I could see that, but this is taken from https://github.com/theforeman/puppet-certs/pull/365/files#diff-9f93cd98c4a64f890a47782b00ef580ea9b901d66db370cd21c399c63991a986R185 and so my intent is not to change this behavior at that level.